packages/collector/test/tracing/protocols/http/native_fetch/test.js

```diff
@@ -64,6 +64,39 @@ mochaSuiteFn('tracing/native fetch', function () {
     await clientControls.clearIpcMessages();
   });
 
+  it('must trace request in background', () => {
+    return clientControls
+      .sendRequest({
+        method: 'GET',
+        path: '/fetch-deferred'
+      })
+      .then(() => {
+        return retry(() => {
+          return globalAgent.instance.getSpans().then(spans => {
+            const entryInClient = verifyRootHttpEntry({
+              spans,
+              host: `localhost:${clientControls.getPort()}`,
+              url: '/fetch-deferred'
+            });
```

Comment:
can we check add a check for no.of spans as welll ?

Reply:
Done